client/src/pages/VerifyEmail.js

```diff
@@ -1,11 +1,107 @@
 import React from "react";
+import { connect } from "react-redux";
+import styled from "styled-components";
+import { mq } from "constants/theme";
 
-const VerifyEmail = () => {
+// ICONS
+import emailVerify from "assets/sign-up-email-verify.svg";
+
+const VerifyEmailContainer = styled.div`
+  display: flex;
+  align-items: center;
+  background-color: #fbfbfd;
+  @media screen and (max-width: ${mq.phone.wide.maxWidth}) {
+    padding-top: 6vh;
+  }
+`;
+
+const VerifyEmailLeftContainer = styled.div`
+  flex-basis: 45%;
+  background-color: #f3f4fe;
+  height: 100vh;
+  position: relative;
+  @media screen and (max-width: ${mq.phone.wide.maxWidth}) {
+    display: none;
+  }
+
+  @media screen and (max-width: ${mq.tablet.narrow.maxWidth}) {
+    flex-basis: 30%;
+  }
+`;
+
+const EnvelopeImageContainer = styled.div`
+  position: absolute;
+  top: 50%;
+  left: 55%;
+  transform: translate(-50%, -50%);
+  width: 80%;
+  margin: 0 auto;
+  @media screen and (max-width: ${mq.tablet.narrow.maxWidth}) {
+    .EnvelopeSVG {
+      width: 100%;
+    }
+  }
+`;
+
+const VerifyEmailRightContainer = styled.div`
+  flex: 1;
+`;
+
+const VerifyDirectionsContainer = styled.div`
+  .directions-header {
+    font-family: Poppins;
+    font-size: 26px;
+    font-weight: bold;
+    font-stretch: normal;
+    font-style: normal;
+    line-height: 1.23;
+    letter-spacing: normal;
+  }
+
+  .directions-text {
+    width: 406px;
+    font-family: Poppins;
+    font-size: 14px;
+    font-weight: 500;
+    font-stretch: normal;
+    font-style: normal;
+    line-height: 1.29;
+    letter-spacing: normal;
+    margin-left: 25%;
+
+    @media screen and (max-width: ${mq.tablet.narrow.maxWidth}) {
+      width: 100%;
+      margin: 0;
+    }
+  }
+`;
+
+const VerifyEmail = ({ email }) => {
   return (
-    <div className="text-center">
-      <h1>Please verify your email</h1>
-    </div>
+    <VerifyEmailContainer>
+      <VerifyEmailLeftContainer>
+        <EnvelopeImageContainer>
+          <img className="EnvelopeSVG" src={emailVerify} alt="" />
+        </EnvelopeImageContainer>
+      </VerifyEmailLeftContainer>
+      <VerifyEmailRightContainer>
+        <div className="text-center">
+          <VerifyDirectionsContainer>
+            <h1 className="directions-header">Check your inbox</h1>
+            <p className="directions-text">
+              We just emailed a link to {email}. Click the link to confirm your
+              account and complete your profile. If you don't see a message
+              within a few minutes please check your spam folder.
+            </p>
+          </VerifyDirectionsContainer>
+        </div>
+      </VerifyEmailRightContainer>
+    </VerifyEmailContainer>
   );
 };
 
-export default VerifyEmail;
+const mapStateToProps = ({ state }) => ({
```

Comment:
I get this error as is so I think this needs to be `session` instead of `state`

![image](https://user-images.githubusercontent.com/10546720/85464892-2a612780-b576-11ea-8a98-74edacce7386.png)

```suggestion
const mapStateToProps = ({ session }) => ({
```





```suggestion
const mapStateToProps = ({ session }) => ({
```

Reply:
Yup same error on my end.

Reply:
Updates the connection to session in commit 427850d

Reply:
`isAuthenticated` simply checks if user has logged in and got a token (which is now also stored in a cookie but I don't think this is related here, unless somehow loading the cookie doesn't load the other data such as `emailVerified` which means that has to be fixed somehow).
but there are additional checks to do, if the email verification check is not handled correctly it might be related to the value in the redux store, is this fixed now @joshmorel ?